src/Console/Application.php

```diff
@@ -15,6 +15,7 @@
 use PhpCsFixer\Console\Command\DescribeCommand;
```

Comment:
I'm not sure whether the case of not being able to use the cache is strong enough to have build-in support for workaround, that we need to maintain in the codebase.
what do you think, @localheinz @julienfalque @kubawerlos @SpacePossum ?

Reply:
@keradus 

On one hand, if anyone cannot use the cache in a CI environment, I would argue that maybe something is wrong with the CI setup. On the other hand, this command is small, so I would be fine with adding it!

Reply:
in case I get your "go", I will add tests etc.

Reply:
The `xargs` solution might be the best we got for a while for multi threading (until we get (better) multi threading for PHP ootb/better 3rd party).
 
If supporting all it takes is the new command `as is` + additional the cache check than I would be in favor (with the cache check support added, documentation, etc. all is on par with project requirements)